libclamav/ole2_extract.c

```diff
@@ -1331,7 +1317,6 @@ static cl_error_t handler_enum(ole2_header_t *hdr, property_t *prop, const char
                     }
                 } while (0);
 
-                free(hwp_check);
```

Comment:
Oh good call, we don't have to free it up here because it's not within the loop. 👍